src/runners/http_service_runner.ts

```diff
@@ -28,41 +28,56 @@ process.on('unhandledRejection', err => {
     }
 });
 
+if (require.main === module) {
+    (async () => {
+        const provider = providerUtils.createWeb3Provider(defaultConfig.ETHEREUM_RPC_URL);
+        const dependencies = await getDefaultAppDependenciesAsync(provider, defaultConfig);
+        await runHttpServiceAsync(dependencies, defaultConfig);
+    })().catch(error => logger.error(error));
+}
+
 /**
  * This service handles the HTTP requests. This involves fetching from the database
  * as well as adding orders to mesh.
+ * @param dependencies If no mesh client is supplied, the HTTP service will start without it.
+ *                     It will provide defaults for other params.
  */
-(async () => {
-    const connection = await getDBConnectionAsync();
-    const app = express();
+export async function runHttpServiceAsync(
+    dependencies: AppDependencies,
+    config: { HTTP_PORT: string },
+    _app?: core.Express,
+): Promise<Server> {
+    const app = _app || express();
     app.use(requestLogger());
+    app.use(cors());
+    app.use(bodyParser.json());
+
     const server = app.listen(config.HTTP_PORT, () => {
-        logger.info(`API (HTTP) listening on port ${config.HTTP_PORT}!\nConfig: ${JSON.stringify(config, null, 2)}`);
+        logger.info(`API (HTTP) listening on port ${config.HTTP_PORT}!`);
     });
-    const stakingDataService = new StakingDataService(connection);
-    // tslint:disable-next-line:no-unused-expression
-    new StakingHttpService(app, stakingDataService);
-    let meshClient;
-    try {
-        meshClient = new WSClient(config.MESH_WEBSOCKET_URI);
-        // tslint:disable-next-line:no-unused-expression
-        new WebsocketService(server, meshClient);
-    } catch (err) {
-        logger.error(err);
+
+    // staking http service
+    app.use(STAKING_PATH, createStakingRouter(dependencies.stakingDataService));
+
+    // SRA http service
+    app.use(SRA_PATH, createSRARouter(dependencies.orderBookService));
+
+    // swap/quote http service
+    if (dependencies.swapService) {
+        app.use(SWAP_PATH, createSwapRouter(dependencies.swapService));
+    } else {
+        logger.warn(`API running without swap service`);
```

Comment:
`logger.error` ?

Reply:
It was warn before but I can change it to error :)